src/docs/environment/index.mdx

```diff
@@ -267,6 +267,12 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   displayed correctly.
 </Alert>
 
+### A Note on Relay
+
+Some services are not run in all situations, and one of those is <Link to="/services/relay/">Relay</Link>. If you need a more production-like environment
+in development, you can set `SENTRY_USE_RELAY=True` in `~/.sentry/sentry.conf.py`. This will launch relay as part of the
+`devserver` workflow.
```

Comment:
Suggestion as per https://github.com/getsentry/sentry/pull/32497#issuecomment-1064419810:

```suggestion
### Ingestion Pipeline (Relay)

Some services are not run in all situations, among those are <Link to="/services/relay/">Relay</Link> and the ingest workers. If you need a more production-like environment
in development, you can set `SENTRY_USE_RELAY=True` in `~/.sentry/sentry.conf.py`. This will launch Relay as part of the
`devserver` workflow.
```